doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -419,11 +419,13 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
    ACRN supports three channels: ``ioc``, ``power button`` and ``uart``.
 
+   For ``uart``, an additional option, ``,allow_trigger_s5``, can be added.
```

Comment:
Bringing @gvancuts comment on PR #6044 to this PR:  Can you also add a short explanation of what that parameter means?